src/openlcb/MemoryConfigClient.cxxtest

```diff
@@ -117,6 +117,13 @@ TEST_F(MemoryConfigClientTest, readsmallpart)
                      b->data()->payload.size()));
 }
 
+TEST_F(MemoryConfigClientTest, unsolicited)
```

Comment:
Please add a comment to describe what required behavior this is testing. I wasn't able to figure that out looking at the test.

Reply:
done